### PR TITLE
Feature-IOPS-1829-Extension Comparisons

### PR DESCRIPTION
**Pull request by KazeemHamzat**
This PR only has Extension Comparisons comparisons issue. Fixing inconsistencies issues in context has been stepped down for DA review.

### CODE REVIEW

File: structuredefinitions/England-Location.xml [hunk 1]
```diff
@@ -40,7 +40,7 @@
     <element id="Location.identifier:odsCode">
       <path value="Location.identifier" />
       <sliceName value="odsCode" />
-      <max value="0" />
+      <max value="1" />
```
Comment (digitalhealthmartin) [inline structuredefinitions/England-Location.xml]: The max is * in NHSD, why just 1?

**Review by digitalhealthmartin — CHANGES_REQUESTED** [review]
Considering this PR contains a lot/all of the Forge recommended changes for Profiles as it is done in the other PR, I think it's best we sort of the other one first, merge it in, then merge it into this branch, and then we should only see and review changes to Extensions as that is the scope of this PR

File: valuesets/ValueSet-England-ChildProtectionPlan.xml
Comment (digitalhealthmartin) [inline valuesets/ValueSet-England-ChildProtectionPlan.xml:1]: Why is ValueSet ChildProtectionPlan here? Surely this PR should just be about the updates to Extensions

**Review by digitalhealthmartin — CHANGES_REQUESTED** [review]
Just a comment about the valueset

File: valuesets/ValueSet-England-ChildProtectionPlan.xml
Comment (KazeemHamzat) [reply valuesets/ValueSet-England-ChildProtectionPlan.xml:1]: The merging of stu1_develop into the branch, Feature-IOPS-1829-Extension-Comparisons-and-Profile-Context-Inconsistencies to resolve conflict introduced it. It ought to come in as a new file but as I have lost my VM, while re-cloning the Repo on my laptop, Git flagged it for me to commit locally to stu1_develop before I can switch branch. Then, pulling stu1_develop into the branch further downloaded it as a new file. The concern will be that only 1 file should exist for ValueSet-ChildProtectionPlan and no changes made to the existing file in stu1_develop.

**Review by digitalhealthmartin — APPROVED** [review]
Based on fact that I put the valueset through a diff tool and both are identical, and GitHub is saying it's a 'modification' and not a new file, I'm presuming it should be fine. Approving.

**Review by SheenaSharma — APPROVED** [review]

**Pull request merged**